tests/unit/src/test/scala/tests/SbtSuite.scala

```diff
@@ -0,0 +1,82 @@
+package tests
+
+class SbtSuite extends BaseLspSuite("sbt") with ScriptsAssertions {
+
+  val scalaVersion = "2.12.10"
```

Comment:
We can use BuildInfo, let's add to imports:
`import scala.meta.internal.metals.{BuildInfo => V}`

and then:
```suggestion
  val scalaVersion = V.scala212
```